test/integration/index.js

```diff
@@ -5,18 +5,22 @@ const logger = require('./logger');
 const config = require('../knexfile');
 const fs = require('fs');
 
-const Bluebird = require('bluebird');
-
 Object.keys(config).forEach((dialectName) => {
   return require('./suite')(logger(knex(config[dialectName])));
 });
 
-after(function(done) {
+before(function() {
+  if (config.sqlite3 && config.sqlite3.connection.filename !== ':memory:') {
+    fs.copyFileSync(
+      __dirname + '/../multilineCreateMasterSample.sqlite3',
```

Comment:
what's the difference between the two files?

Reply:
I had to make a copy before the test run and delete it after because knex migration tables changes, so the DB file becomes changed in Git for every test run.